packages/front-end/tests/e2e/org-admin/Labs_Org_Admin.spec.e2e.ts

```diff
@@ -1,34 +1,170 @@
 import { test, expect } from 'playwright/test';
+//const invitemail = 'marvin.umali+eg99@deptagency.com';
+//const orgName = 'Automated Org';
```

Comment:
Is there a different email that can be added here rather than your personal one? 

Also may be better to add these as // TODOs with an accompanying comment about the intent

Reply:
Unfortunately we don't have a group email yet. Maybe it has to be requested so that few members will receive an email (e.g. team@easygenomics.org).
OK will add to do for the comments